include/ClientConfig.h

```diff
@@ -36,23 +36,32 @@ class ClientConfig {
     ClientConfig(
             std::string username = std::string(""),
             std::string password = std::string(""),
-            ClientAuthHashScheme scheme = HASH_SHA1, bool enableAbandon = false);
+            ClientAuthHashScheme scheme = HASH_SHA1, bool enableAbandon = false,
+            bool enableQueryTimeout = false, timeval tv = (timeval) {DEFAULT_QUERY_TIMEOUT_SEC, 0});
```

Comment:
If we want to define timeout in seconds only and no discrete usec, we can simpilfy it here and have client consume only seconds with usec being default.